### DISCUSSION TITLE
How to switch to dev in windows

### DISCUSSION THREAD

**Discussion by Burki24** [Koenkk/zigbee2mqtt#10399] [General]
Hi @Koenkk ,

I've a short question regarding switching from master to dev in Windows environment. Is that the same way as in Linux?
I didn't work with windows and zigbee2mqtt, but a friend asked me as my RPI-Installation supports devices which are not integrated in the last master-Version Konverter: Master:14.0.336 / Dev:14.0.360)

It would be very nice, if someone can give me a small hint.
Thanks in advance

Burkhard

**Comment by Burki24** [comment]
No one here, who can help?

**Reply by Koenkk** [reply]
The instructions below "Update" should work on Windows: https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html#linux
